diff --git a/node_modules/@react-native-firebase/app-check/ios/RNFBAppCheck/RNFBAppCheckProvider.m b/node_modules/@react-native-firebase/app-check/ios/RNFBAppCheck/RNFBAppCheckProvider.m
index 6722366..44f3f2f 100644
--- a/node_modules/@react-native-firebase/app-check/ios/RNFBAppCheck/RNFBAppCheckProvider.m
+++ b/node_modules/@react-native-firebase/app-check/ios/RNFBAppCheck/RNFBAppCheckProvider.m
@@ -83,7 +83,7 @@
 - (void)getLimitedUseTokenWithCompletion:(nonnull void (^)(FIRAppCheckToken *_Nullable,
                                                            NSError *_Nullable))handler {
   DLog(@"proxying getLimitedUseTokenWithCompletion to delegateProvider...");
-  [self.delegateProvider getLimitedUseTokenWithCompletion:handler];
+//  [self.delegateProvider getLimitedUseTokenWithCompletion:handler];
 }
 
 @end
